linalg/complex_operator.hpp

```diff
@@ -168,6 +174,69 @@ class ComplexSparseMatrix : public ComplexOperator
    virtual Type GetType() const { return MFEM_ComplexSparseMat; }
 };
 
+#ifdef MFEM_USE_SUITESPARSE
+/** @brief Interface with UMFPack solver specialized for ComplexSparseMatrix
+    This approach avoids forming a monolithic SparseMatrix which leads
+    to increased memory and flops
+ */
+class ComplexUMFPackSolver : public Solver
+{
+protected:
+   bool use_long_ints;
+   bool transa = false;
```

Comment:
Just a minor suggestion: you could add this as an optional argument (defaulting to `false)` to the constructors (instead of setting it to `false` here) so that `transa` can be set without the additional call to `SetTransposeSolve`.

Reply:
Good idea. Done!